StonehearthEditor/pages/styles/main.css

```diff
@@ -41,34 +39,87 @@
     left: 470px;
     width: 30px;
     text-align: center;
-    margin-top: -3px;
+    margin-top: 0px;
 }
+
+.origin-parameter-property {
+    background-color: #ababab;
+    border: 5px solid #9c9c9c;
+    padding: 5px;
+}
+
+.burst-parameter-property {
+    background-color: #ababab;
+    border: 5px solid #9c9c9c;
+    padding: 5px;
+}
+.burst-parameter-property-header-text {
+    width: 110px;
+    display: inline-block;
+    text-align: left;
+    margin-left: 15px;
+}
+
+.level2-parameter-property {
+    background-color: #ababab;
+    border: 5px solid #9c9c9c;
+    padding: 5px;
+}
+
 .level3-parameter-property {
     margin: 7px 0px 7px 5px;
     min-height: 38px;
+    background-color: #ababab;
+    border: 5px solid #9c9c9c;
+    padding: 5px;
 }
 .level3-parameter-property-expand-button {
     position: absolute;
     left: 470px;
     width: 30px;
     text-align: center;
-    margin-top: -3px;
+    margin-top: 0px;
 }
 .level3-parameter-property-header-text {
     width: 110px;
     display: inline-block;
+    text-align: left;
+    margin-left: 15px;
 }
 .level3-parameter-property-select {
     width: 250px;
 }
-.parameter-property-label {
+.origin-parameter-property-header-text {
     width: 110px;
     display: inline-block;
+    text-align: left;
+    margin-left: 15px;
+}
+.origin-parameter-property-select {
+    width: 250px;
+}
+.parameter-property-label {
+    width: 125px;
+    display: inline-block;
     text-align: right;
     font-style: italic;
 }
+.parameter-property-label-color {
+    display: inline-block;
+    text-align: center;
+}
+.inline-property-label {
+    width: 135px;
+    display: inline-block;
+    text-align: center;
+}
 .sh-numeric-input {
-    width: 100px;
+    width: 75px;
+}
+.sh-material-input {
+    width: 400px;
+}
+.material-warning {
```

Comment:
this class is empty, maybe we don't need it.

Reply:
removed